lib/check/do_single_expression.ex

```diff
@@ -0,0 +1,89 @@
+defmodule CompassCredoPlugin.Check.DoSingleExpression do
+  use Credo.Check,
+    base_priority: :normal,
+    category: :readability,
+    explanations: [
+      check: """
+      Use do: for single line if/unless statements
+
+      Prefer using the single-line function as long as mix format command satisfies to keep the function body just in one line.
+
+      # Less Preferred
+      if some_condition do
+        "some_stuff"
+      end
+
+      def validate_coupon() do
+        :ok
+      end
+
+      def create_voucher(attrs \\ %{}),
```

Comment:
I think we can remove this one from the documentation since we are not handling this case (and one example below as well).

Reply:
[2367799](https://github.com/nimblehq/compass-credo-plugin/pull/14/commits/2367799674ac0580494288afbc66a8589222b723)